tests/main/install-fontconfig-cache-gen/task.yaml

```diff
@@ -10,7 +10,7 @@ debug: |
     ls -lah /var/cache/fontconfig/
 
 prepare: |
-    if [ "$SPREAD_SYSTEM" = ubuntu-16.04-64 ]; then
+    if os.query is-xenial; then
```

Comment:
won't this now match on ubuntu-16.04-32 where it wouldn't before? maybe it's fine for this specific test to also run on xenial 32-bit, but in general I think we might need silly helpers like `os.query is-xenial32` and `is-xenial64` and `is-xenial` (for either 32 or 64 bit)

Reply:
hi @anonymouse64, thanks for the review, this test just run 64bits so it should be ok the change.

About to have is-xenial32 and is-xenial64, it is the next step, also will be added commands for some versions of other OSs like opensuse-tumbleweed.

Reply:
In this case this should have a TODO so we don't forget to revisit affected tests?

Reply:
Sorry, ignore my question, and I got confused.